contribs/gmf/src/controllers/abstract.js

```diff
@@ -65,11 +65,12 @@ gmf.module.value('ngeoQueryOptions', {
  * @param {gmfx.Config} config A part of the application config.
  * @param {angular.Scope} $scope Scope.
  * @param {angular.$injector} $injector Main injector.
+ * @param {angular.$timeout} $timeout Angular timeout service.
  * @constructor
  * @ngInject
  * @export
  */
-gmf.AbstractController = function(config, $scope, $injector) {
+gmf.AbstractController = function(config, $scope, $injector, $timeout) {
```

Comment:
We use the $injector to avoid all those changes that affect also the client project ...